src/models/consent.ts

```diff
@@ -40,18 +40,28 @@ export enum ConsentStatus {
    * Waiting for the user to confirm payee information and provide more
    * details about the transaction.
    */
-  PENDING_PAYEE_CONFIRMATION = 'PENDING_PAYEE_CONFIRMATION',
+  PENDING_PARTY_CONFIRMATION = 'PENDING_PARTY_CONFIRMATION',
```

Comment:
I would keep PAYEE if it is more related to usage, PARTY is much more generic

Reply:
I think this should be fine since this change is based on the sequence diagrams [here](https://github.com/mojaloop/pisp-demo-server/blob/master/docs/design/linking.md). Also, there's no payee involved when establishing a consent, so I suspect that the current code in the master is problematic... probably due to copying the enum used for the transfer flow before